sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -125,22 +126,18 @@ public HttpPipeline getHttpPipeline() {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<String>> createRelationshipWithResponse(String digitalTwinId, String relationshipId, String relationship) {
```

Comment:
won't we be having the same issue with the headers if we use the default Response type?

Reply:
Yup, that is correct. This snippet is meant for testing, the actual impl is in #14495 .
 
The actual API surface is:
```java
public Mono<DigitalTwinsResponse<String>> createRelationshipWithResponse(String digitalTwinId, String relationshipId, String relationship) { }
```